lib/boorus/moebooru/pages/moebooru_home_page.dart

```diff
@@ -101,10 +102,19 @@ class _MoebooruHomePageState extends ConsumerState<MoebooruHomePage> {
           icon: Symbols.local_fire_department,
           title: 'Hot',
         ),
+          HomeNavigationTile(
+            value: 3,
+            constraints: constraints,
+            selectedIcon: Symbols.favorite,
+            icon: Symbols.favorite,
+            title: 'Favorites',
+          ),
       ],
-      desktopViews: const [
+      desktopViews: [
         MoebooruPopularPage(),
         MoebooruPopularRecentPage(),
+        ref.watchBooruBuilder(ref.watchConfig)!
```

Comment:
Don't use bang operator. Refer to `gelbooru_home_page.dart` for how to handle null for this case.